lib/ansible/plugins/httpapi/checkpoint.py

```diff
@@ -33,20 +33,23 @@ class HttpApi(HttpApiBase):
     def login(self, username, password):
         if username and password:
             payload = {'user': username, 'password': password}
-            url = '/web_api/login'
+            CONTEXT = os.environ.get('CONTEXT', 'web_api')
```

Comment:
may I know the reason of this change

Reply:
We use the same connection (`httpapi`), but for gaia APIs the url should be 'gaia_api' instead of 'web_api'. So I didn't find better solution than set `CONTEXT` environment variable that will have the value 'gaia_api' when we want to use it for gaia modules.
If we don't have it, the value will stay 'web_api'.

Tell me if there is better solution - like passing it as a parameter from the module code. Thanks.

Reply:
@chkp-yuvalfe ideally based on module either `web_api`/`gaia_api` should be called, as both the modules are named as `cp_mgmt_*`/`cp_gaia_*`, module should be intelligent enough to call the respective api and user intervention shouldn't be needed either as module input or setting env var.